.shippable.yml

```diff
@@ -61,6 +61,7 @@ build:
     - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_TA_GPROF_SUPPORT=y CFG_FTRACE_SUPPORT=y CFG_SYSCALL_FTRACE=y CFG_ULIBS_MCOUNT=y
     - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_VIRTUALIZATION=y
     - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_CORE_SEL1_SPMC=y
+    - dd if=/dev/urandom of=BL32_AP_MM.fd bs=2621440 count=1 && _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_STMM_PATH=BL32_AP_MM.fd CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=1 CFG_CORE_HEAP_SIZE=524288 CFG_TEE_RAM_VA_SIZE=0x00400000
```

Comment:
One minor thing: are these parameters needed? `CFG_RPMB_FS_DEV_ID=1 CFG_CORE_HEAP_SIZE=524288 CFG_TEE_RAM_VA_SIZE=0x00400000`
- The first one is clearly used at runtime only, and this is a compile test so I think it could be removed
- The two other ones could perhaps be set by default when `CFG_WITH_SECURE_PARTITION` is set? (it looks like the default values cannot work in which case which would mean they're not good defaults ;-)).

Reply:
The first one is needed for QEMU, if we ever intend on doing runtime tests, but can be removed for now.
I'll switch over the other two as defaults and squash push the patches

Reply:
Sounds good. You can also remove `CFG_ARM64_core=y` since [it is now the default for this platform](https://github.com/OP-TEE/optee_os/commit/b2aee6035f4e573d9a946f8747df66c1d9781e14).

Reply:
@jforissier on a second thought, CFG_CORE_HEAP_SIZE=524288  is only required when using an RPMB backend.
Although that's the only option for now, we do have code using a secure NOR flash, that we intend to upstream.
So I am not entirely sure defaulting the increased heap size is a good idea.
On the other hand you can always override it, so up to you

Reply:
I'm fine with keeping the options in the Shippable file if you'd rather not have them in the default config.

Reply:
Great, it makes more sense in the long run. I'll rebase, squash and remove CFG_RPMB_FS_DEV_ID